bots/naughty/fedora-28/9328-selinux-chronyd-socket

```diff
@@ -0,0 +1 @@
+Error: audit: type=1400 audit(*): avc:  denied  { sendto } for * comm="chronyd" path="/run/chrony/chrony*.sock"
```

Comment:
It looks like this one has been fixed downstream already.  Should we just rebuild the image?

Reply:
OK.